solidity/contracts/Router.sol

```diff
@@ -73,6 +73,20 @@ abstract contract Router is AbacusConnectionClient, IMessageRecipient {
         _enrollRemoteRouter(_domain, _router);
     }
 
+    /**
+     * @notice Batch version of `enrollRemoteRouter`
+     * @param _domains The domaisn of the remote Application Routers
+     * @param _routers The addresses of the remote Application Routers
+     */
+    function enrollRemoteRouters(
+        uint32[] calldata _domains,
+        bytes32[] calldata _routers
+    ) external virtual onlyOwner {
```

Comment:
require lengths to be equal or alternatively require `struct { domain, address }`
imo the current interface is error prone